https://issues.redhat.com/browse/ACM-21420

dockerymick opened the pull request.

dockerymick committed:
https://issues.redhat.com/browse/ACM-21420

subbarao-meduri commented on observability/observability_alerts.adoc:
```diff
@@ -1,20 +1,27 @@
 = Managing alerts
 
-Receive and define alerts for the observability service to be notified of hub cluster and managed cluster changes.
+Receive and define alerts as notifications for any Observability service changes in your hub cluster and managed cluster.
```
This new revision seems to be even more confusing. Here is a suggestion

Configure observability alerts to monitor both hub and managed cluster activity, forward platform and user workload alerts to the hub Alertmanager, and route notifications to external systems such as Slack, email, or PagerDuty.

dockerymick replied:
```suggestion
Configure Observability alerts to monitor both your hub cluster and managed cluster activity, forward platform and user workload alerts to the Alertmanager on your hub cluster, and route notifications to external systems such as Slack, email, or PagerDuty.
```

subbarao-meduri commented on observability/observability_alerts.adoc:
```diff
@@ -1,20 +1,27 @@
 
 - <<alert-prereqs,Prerequisites>>
 - <<configuring-alertmanager,Configuring Alertmanager>>
+- <<mount-secrets-alertmanager,Mounting secrets within the Alertmanager pods>>
```
Can we retitle this as well as mention the real use case in the details section? Mounting secrets is an action for a specific use case. 

The real purpose of this feature is to enable secure external communication from AlertManager to third party end points such as email, slack, pagerduty etc. The goal is to keep the credentials secure and manageable through Kubernetes Secret resources.

dockerymick replied:
```suggestion
- <<secure-comm-alertmanager,Securing communication from Alertmanager to third-party endpoints>>
```

subbarao-meduri commented on observability/observability_alerts.adoc:
```diff
@@ -1,20 +1,27 @@
+- You must enable Observability on your hub cluster.
+- You must have the create permission for secret resources in the `open-cluster-management-observability` namespace.
+- You must have edit permission on the `MultiClusterObservability` resource.
+- You must configure the Prometheus instances on your managed cluster to help prevent metric loss during restart. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/configuring-core-platform-monitoring#configuring-persistent-storage_storing-and-recording-data[Configuring persistent storage] in the {ocp-short} documentation.
```
Perhaps point to 4.19 doc reference? I am not sure what the best practice here is.

dockerymick replied:
ACM/MCE support n-2 versions for OCP, so it is best to link to the minimum supported version. We do this across the doc with the exception of some components.

subbarao-meduri commented on observability/observability_alerts.adoc:
```diff
@@ -61,12 +68,12 @@ route:
 
 [#mount-secrets-alertmanager]
-=== Mounting secrets within the Alertmanager pods
+== Mounting secrets within the Alertmanager pods
```
See comment above. I think we should call out the use case. Secure communication from AlertManager to third party endpoints while keeping the credentials secure and manageable through Kubernetes Secret resources.

dockerymick replied:
```suggestion
== Securing communication from Alertmanager to third-party endpoints
```

subbarao-meduri commented on observability/observability_alerts.adoc:
```diff
@@ -170,10 +177,25 @@ global:
     proxy_url: http://****
 ----
 
+To forward user workload alerts, the alerts must be processed by user workload Prometheus instance and not by Thanos ruler. See the following example of the Prometheus instance:
```
The example is for a `PrometheusRule` resource. Not Prometheus instance.

dockerymick replied:
```suggestion
To forward user workload alerts, the alerts must be processed by the user workload Prometheus instance and not by Thanos ruler. See the following example of the `PrometheusRule` resource:
```

subbarao-meduri commented on observability/observability_alerts.adoc:
```diff
@@ -182,14 +204,28 @@ metadata:
-When you set the annotation, the alert forwarding configuration on the managed clusters is reverted. Any changes made to the `ocp-monitoring-config` config map in the `openshift-monitoring` namespace are also reverted. Setting the annotation ensures that the `ocp-monitoring-config` config map is no longer managed or updated by the observability operator endpoint. After you update the configuratio ...
-
-*Important:* Metrics on your managed cluster are lost if you have a Prometheus instance with a persistent volume for metrics, and the Prometheus instance restarts. Metrics from the hub cluster are not affected.
+*Important:* You lose metrics on your managed cluster if you have a Prometheus instance with a persistent volume for metrics, and the Prometheus instance restarts. Metrics from the hub cluster are not affected.
```
This statement is not accurate. Metrics at the managed cluster Prometheus are lost if Prometheus instance is **not** configured with a persistent volume. (By default, OCP Prometheus instances are not configured with persistent volume.)

dockerymick replied:
```suggestion
*Important:* You lose metrics when Prometheus on your managed cluster is not configured with a persistent volume.
```

subbarao-meduri commented on observability/observability_alerts.adoc:
```diff
@@ -182,14 +204,28 @@ metadata:
         mco-disable-alerting: "true"
 ----
```
The yaml indentation seems to be off. 

Perhaps it make more sense to also provide a script snippet on how to set this annotation?
```
oc annotate MultiClusterObservability observability mco-disable-uwm-alerting=true
```

subbarao-meduri commented on observability/observability_alerts.adoc:
```diff
@@ -182,14 +204,28 @@ metadata:
+metadata:
+  annotations:
+    mco-disable-uwl-alerting: "true"
+----
```
Same as above. Perhaps also provide a code snippet?

```
oc annotate MultiClusterObservability observability mco-disable-uwl-alerting=true
```

dockerymick replied:
```suggestion
Run the following command to set the `mco-disable-uwl-alerting` annotation to `"true"`:

[source,bash]
----
oc annotate MultiClusterObservability observability mco-disable-uwl-alerting=true
----
```

subbarao-meduri reviewed (changes requested):
@dockerymick please let me know if you like to discuss these review comments.

dockerymick commented on observability/observability_alerts.adoc:
```diff
@@ -61,12 +68,12 @@ route:
-Your changes are applied immediately after it is modified. For an example of Alertmanager, see link:https://github.com/prometheus/alertmanager/blob/main/doc/examples/simple.yml[prometheus/alertmanager].
+After you modify the secret, your changes are applied immediately. For an example of Alertmanager, see link:https://github.com/prometheus/alertmanager/blob/main/doc/examples/simple.yml[prometheus/alertmanager].
 
 [#mount-secrets-alertmanager]
```
```suggestion
[#secure-comm-alertmanager]
```

dockerymick commented on observability/observability_alerts.adoc:
```diff
@@ -61,12 +68,12 @@ route:
+== Mounting secrets within the Alertmanager pods
 
-You can create `Secret` resources with arbitrary content, which can be mounted within your `alertmanager` pods for access to authorization credentials.
+You can create `Secret` resources with arbitrary content that you can mount within your `alertmanager` pods for access to authorization credentials.
```
```suggestion
Enable secure external communication from Alertmanager to third-party endpoints such as Slack, email, and PagerDuty by keeping your credentials secure and manageable through Kubernetes `Secret` resources. You can create `Secret` resources with arbitrary content that you can mount within your `alertmanager` pods for access to authorization credentials.
```

dockerymick committed:
Apply suggestions from code review

dockerymick commented on observability/observability_alerts.adoc:
```diff
@@ -170,10 +177,25 @@ global:
-To disable alert forwarding for managed clusters, add the following annotation to the `MultiClusterObservability` custom resource:
+Configuration updates to the `ocp-monitoring-config` config map in the `openshift-monitoring` namespace are reverted. Setting the annotation ensures that the `ocp-monitoring-config` config map is not managed or updated by the Observability operator endpoint. After you update the configuration, both platform and user workload Prometheus instances on your managed cluster restarts.
+
+See the following example where the `mco-disable-alerting` annotation is set to `"true"`:
```
```suggestion
Run the following command to  set the `mco-disable-alerting` annotation to `"true"`:

[source,bash]
----
oc annotate MultiClusterObservability observability mco-disable-uwm-alerting=true
----
```

dockerymick committed:
Update observability/observability_alerts.adoc

dockerymick committed:
Update observability_alerts.adoc

dockerymick commented on observability/observability_alerts.adoc:
```diff
@@ -140,7 +147,7 @@ oc -n open-cluster-management-observability replace secret --filename=-
 == Forwarding alerts
 
-After you enable observability, alerts from your {ocp-short} managed clusters are automatically sent to the hub cluster. You can use the `alertmanager-config` YAML file to configure alerts with an external notification system. 
+After you enable observability, alerts from your {ocp-short} managed clusters are automatically sent to Alertmanager on your hub cluster. By default, all platform alerts are sent to Alertmanager on your hub cluster. When you enable user workload alerts on your {ocp-short} managed clusters, user workload alerts are also sent to your hub cluster. You can use the `alertmanager-config` YAML file to c ...
```
```suggestion
After you enable Observability, alerts from your {ocp-short} managed clusters are automatically sent to Alertmanager on your hub cluster. By default, all platform alerts are sent to Alertmanager on your hub cluster. When you enable user workload alerts on your {ocp-short} managed clusters, user workload alerts are also sent to your hub cluster. You can use the `alertmanager-config` YAML file to configure alerts with an external notification system.
```

dockerymick committed:
Update observability/observability_alerts.adoc

dockerymick committed:
Adding what's new entry

dockerymick committed:
Fixing merge conflicts

dockerymick committed:
Merge branch '2.14_stage' into mj-ACM-21420-214

jc-berger commented on observability/observability_alerts.adoc:
```diff
@@ -1,20 +1,27 @@
 = Managing alerts
 
-Receive and define alerts for the observability service to be notified of hub cluster and managed cluster changes.
+Configure Observability alerts to monitor both your hub cluster and managed cluster activity, forward platform and user workload alerts to the Alertmanager on your hub cluster, and route notifications to external systems such as Slack, email, or PagerDuty.
```
little long of a sentence, and although I understood when I read it fully, at first, I thought we were forwarding the managed cluster activity to a platform or user workload. What do you think about something like this? 

```suggestion
Configure Observability alerts to have the following capabilities: 

- Monitor both your hub cluster and managed cluster activity.
-  Forward platform and user workload alerts to the Alertmanager on your hub cluster.
-  Route notifications to external systems such as Slack, email, or PagerDuty.

```

jc-berger replied:
That intro sentence can actually take lots of different forms, based off what you think is best. For example:

Configuring Observability alerts gives you the following capabilities: 

When you configure Observability alerts, you can complete the following actions: 

When you configure Observability alerts, you have the following capabilities:

jc-berger commented on observability/observability_alerts.adoc:
```diff
@@ -1,20 +1,27 @@
-* You must enable observability on your hub cluster.
-* You must have the create permission for secret resources in the `open-cluster-management-observability` namespace.
-* You must have edit permission on the `MultiClusterObservability` resource.
+- You must enable Observability on your hub cluster.
```
optional, I see the original content had the "you must..." structure, so it's cool if you want to keep it. I honestly don't think it's necessary though. What do you think of this? 

```suggestion

- Enable Observability on your hub cluster.
```

jc-berger commented on observability/observability_alerts.adoc:
```diff
@@ -1,20 +1,27 @@
-* You must have the create permission for secret resources in the `open-cluster-management-observability` namespace.
-* You must have edit permission on the `MultiClusterObservability` resource.
+- You must enable Observability on your hub cluster.
+- You must have the create permission for secret resources in the `open-cluster-management-observability` namespace.
```
```suggestion

- Have the create permission for `Secret` resources in the `open-cluster-management-observability` namespace.
```

jc-berger commented on observability/observability_alerts.adoc:
```diff
@@ -1,20 +1,27 @@
-* You must have edit permission on the `MultiClusterObservability` resource.
+- You must enable Observability on your hub cluster.
+- You must have the create permission for secret resources in the `open-cluster-management-observability` namespace.
+- You must have edit permission on the `MultiClusterObservability` resource.
```
```suggestion

- Have edit permission on the `MultiClusterObservability` resource.
```

jc-berger commented on observability/observability_alerts.adoc:
```diff
@@ -1,20 +1,27 @@
+- You must enable Observability on your hub cluster.
+- You must have the create permission for secret resources in the `open-cluster-management-observability` namespace.
+- You must have edit permission on the `MultiClusterObservability` resource.
+- You must configure the Prometheus instances on your managed cluster to help prevent metric loss during restart. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/configuring-core-platform-monitoring#configuring-persistent-storage_storing-and-recording-data[Configuring persistent storage] in the {ocp-short} documentation.
```
```suggestion

- Configure the Prometheus instances on your managed cluster to help prevent metric loss during restart. For help, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/configuring-core-platform-monitoring#configuring-persistent-storage_storing-and-recording-data[Configuring persistent storage] in the {ocp-short} documentation.
```

jc-berger replied:
I know it's a little wordy, but if we give a link in a prereq, we're prompting to the user to follow that link and complete the documentation there. 

Do you want users to use this link to have steps to configure promtheus instances on their managed clusters? Or is it an optional resource should they need it? 

I think if we do something like "for help" that let's users know that if they don't know if they have configured it, they need to go here, but if they know that have configured it, this link isn't necessary for them to click. 

```suggestion

- You must configure the Prometheus instances on your managed cluster to help prevent metric loss during restart. For help, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/configuring-core-platform-monitoring#configuring-persistent-storage_storing-and-recording-data[Configuring persistent storage] in the {ocp-short} documentation.
```

jc-berger commented on observability/observability_alerts.adoc:
```diff
@@ -1,20 +1,27 @@
+- You must have the create permission for secret resources in the `open-cluster-management-observability` namespace.
+- You must have edit permission on the `MultiClusterObservability` resource.
+- You must configure the Prometheus instances on your managed cluster to help prevent metric loss during restart. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/configuring-core-platform-monitoring#configuring-persistent-storage_storing-and-recording-data[Configuring persistent storage] in the {ocp-short} documentation.
+- You must enable user workload monitoring and alerts on your managed clusters. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/configuring-user-workload-monitoring[Configuring user workload monitoring] and link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/managing-alerts[Managi ...
```
```suggestion

- Enable user workload monitoring and alerts on your managed clusters. For help, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/configuring-user-workload-monitoring[Configuring user workload monitoring] and link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/managing-alerts[Managing alerts] in the {ocp-short} documentation.
```

jc-berger replied:
Same logic as above, and once again, optional :) 

```suggestion
- You must enable user workload monitoring and alerts on your managed clusters. For help, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/configuring-user-workload-monitoring[Configuring user workload monitoring] and link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/managing-alerts[Managing alerts] in the {ocp-short} documentation.
```

jc-berger commented on observability/observability_alerts.adoc:
```diff
@@ -1,20 +1,27 @@
+- You must have edit permission on the `MultiClusterObservability` resource.
+- You must configure the Prometheus instances on your managed cluster to help prevent metric loss during restart. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/configuring-core-platform-monitoring#configuring-persistent-storage_storing-and-recording-data[Configuring persistent storage] in the {ocp-short} documentation.
+- You must enable user workload monitoring and alerts on your managed clusters. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/configuring-user-workload-monitoring[Configuring user workload monitoring] and link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/managing-alerts[Managi ...
+- The user workload Prometheus requires adding a label to alerting rules in order to force only user workload Prometheus that are deployed in the `openshift-user-workload-monitoring` namespace to evaluate the alerting rule and prevents the Thanos Ruler instance from processing them. For more details, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitori ...
```
```suggestion

- For the user workload for Prometheus, add a label to alerting rules in order to force only user workload Prometheus that are deployed in the `openshift-user-workload-monitoring` namespace to evaluate the alerting rule and prevents the Thanos Ruler instance from processing them. For help, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/about-openshift-container-platform-monitoring#about-creating-alerting-rules-for-user-defined-projects_key-concepts[Creating alerting rules for user-defined projects] in the {ocp-short} documentation.
```

jc-berger replied:
```suggestion
- The user workload Prometheus requires adding a label to alerting rules in order to force only user workload Prometheus that are deployed in the `openshift-user-workload-monitoring` namespace to evaluate the alerting rule and prevents the Thanos Ruler instance from processing them. For help, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/about-openshift-container-platform-monitoring#about-creating-alerting-rules-for-user-defined-projects_key-concepts[Creating alerting rules for user-defined projects] in the {ocp-short} documentation.
```

jc-berger commented on observability/observability_alerts.adoc:
```diff
@@ -1,20 +1,27 @@
 
 [#alert-prereqs]
 == Prerequisites
 
```
If you want, you could include an intro sentence to this list: 
```suggestion

You must have the following credentials and complete the following actions to manage alerts:
```

jc-berger commented on observability/observability_alerts.adoc:
```diff
@@ -61,12 +68,12 @@ route:
+== Securing communication from Alertmanager to third-party endpoints
 
-You can create `Secret` resources with arbitrary content, which can be mounted within your `alertmanager` pods for access to authorization credentials.
+Enable secure external communication from Alertmanager to third-party endpoints such as Slack, email, and PagerDuty by keeping your credentials secure and manageable through Kubernetes `Secret` resources. You can create `Secret` resources with arbitrary content that you can mount within your `alertmanager` pods for access to authorization credentials.
```
By enabling secure external communication you keep your credentials secure? Or you can only enable secure external communication by keeping the credentials secure? 

Reading this sentence, I'm thinking, we must keep the credentials secure and manageable in order to enable secure external communication. 

Correct me if I am wrong, but I think you mean something more like this? 
```suggestion
Enable secure external communication from Alertmanager to third-party endpoints such as Slack, email, and PagerDuty to keep your credentials secure and manageable through Kubernetes `Secret` resources. You can create `Secret` resources with arbitrary content that you can mount within your `alertmanager` pods for access to authorization credentials.
```

jc-berger commented on observability/observability_alerts.adoc:
```diff
@@ -170,25 +177,51 @@ global:
++
+*Important:* You lose metrics when Prometheus on your managed cluster is not configured with a persistent volume.
++
+When the changes are reverted, a config map named `cluster-monitoring-reverted` is created in the `open-cluster-management-addon-observability` namespace. Any new, alert forward configurations that are manually added are not reverted from the config map. 
```
Could this content go in the intro, before the first step? Essentially, put this paragraph (line 208) on line 198, then provide the first command "Run .... " on line 200? 

I'm not used to seeing this much content in the middle of a procedure unless it's a Note or Important notice (like you have above). If you want, I think this content could even be part of a fairly long "Important" notice.

jc-berger commented on observability/observability_alerts.adoc:
```diff
@@ -170,25 +177,51 @@ global:
++
+When the changes are reverted, a config map named `cluster-monitoring-reverted` is created in the `open-cluster-management-addon-observability` namespace. Any new, alert forward configurations that are manually added are not reverted from the config map. 
+
+. Verify that the hub cluster alert manager is no longer propagating managed cluster alerts to third-party messaging tools. See the previous section, _Configuring Alertmanager_.
```
I don't think we need the last sentence here, and it's kind of self referential. 

```suggestion
. Verify that the hub cluster alert manager is no longer propagating managed cluster alerts to third-party messaging tools. 
```

coleenquadros committed:
Apply suggestions from code review

Enable secure external communication from Alertmanager to third-party endpoints such as Slack, email, and PagerDuty by keeping your credentials secure and manageable through Kubernetes `Secret` resources. You can create `Secret` resources with arbitrary content that you can mount within your `alertmanager` pods for access to authorization credentials.

Co-authored-by: jc-berger <70717303+jc-berger@users.noreply.github.com>

coleenquadros commented on release_notes/acm_new.adoc:
```diff
@@ -105,6 +105,10 @@ For other Application and GitOps topics, see link:../applications/app_management
 
 - *Technology Preview:* Use right-sizing guides to optimize workloads on a cluster and namespace level. To learn more, see link:../observability/obs_right_size.adoc#optimize-work-right-size[Optimizing workloads by using right-sizing guides (Technology Preview)].
 
+- You can disable user workload alert fowarding to Alertmanager for your managed clusters by setting the `mco-disable-uwl-alerting` annotation to `true`. When you set the annotation, user workload alert forwarding to Alertmanager for your hub cluster stops. See link:../observability/observability_alerts.adoc#disabe-uwl-forwarding[Disabling user workload alert forwarding for managed clusters] for  ...
```
```suggestion
- You can disable user workload alert fowarding to Alertmanager for your managed clusters by setting the `mco-disable-uwl-alerting` annotation to `true`. See link:../observability/observability_alerts.adoc#disabe-uwl-forwarding[Disabling user workload alert forwarding for managed clusters] for more details.
```

coleenquadros commented on observability/observability_alerts.adoc:
```diff
@@ -61,12 +68,12 @@ route:
+== Securing communication from Alertmanager to third-party endpoints
 
-You can create `Secret` resources with arbitrary content, which can be mounted within your `alertmanager` pods for access to authorization credentials.
+Enable secure external communication from Alertmanager to third-party endpoints such as Slack, email, and PagerDuty to keep your credentials secure and manageable through Kubernetes `Secret` resources. You can create `Secret` resources with arbitrary content that you can mount within your `alertmanager` pods for access to authorization credentials.
```
```suggestion
Enable secure external communication from Alertmanager to third-party endpoints such as Slack, email, and PagerDuty by keeping your credentials secure and manageable through Kubernetes `Secret` resources. You can create `Secret` resources with arbitrary content that you can mount within your `alertmanager` pods for access to authorization credentials.
```

dockerymick commented on observability/observability_alerts.adoc:
```diff
@@ -170,25 +177,51 @@ global:
+Configuration updates to the `ocp-monitoring-config` config map in the `openshift-monitoring` namespace are reverted. Setting the annotation ensures that the `ocp-monitoring-config` config map is not managed or updated by the Observability operator endpoint. After you update the configuration, both platform and user workload Prometheus instances on your managed cluster restarts. Complete the foll ...
 
-Verify that the hub cluster alert manager is no longer propagating managed cluster alerts to third-party messaging tools. See the previous section, _Configuring Alertmanager_.
+. Run the following command to  set the `mco-disable-alerting` annotation to `"true"`:
```
```suggestion
When the changes are reverted, a config map named `cluster-monitoring-reverted` is created in the `open-cluster-management-addon-observability` namespace. Any new, alert forward configurations that are manually added are not reverted from the config map.

. Run the following command to  set the `mco-disable-alerting` annotation to `"true"`:
```

dockerymick committed:
Apply suggestions from code review

Co-authored-by: jc-berger <70717303+jc-berger@users.noreply.github.com>
Co-authored-by: Coleen Iona Quadros <coleen.quadros27@gmail.com>

dockerymick committed:
Apply suggestions from code review

Co-authored-by: jc-berger <70717303+jc-berger@users.noreply.github.com>

dockerymick commented on observability/observability_alerts.adoc:
```diff
@@ -170,25 +189,53 @@ global:
 ----
++
+*Important:* You lose metrics when Prometheus on your managed cluster is not configured with a persistent volume.
++
+When the changes are reverted, a config map named `cluster-monitoring-reverted` is created in the `open-cluster-management-addon-observability` namespace. Any new, alert forward configurations that are manually added are not reverted from the config map. 
```
```suggestion
```

dockerymick committed:
Apply suggestions from code review

subbarao-meduri commented on observability/observability_alerts.adoc:
```diff
@@ -170,25 +189,51 @@ global:
+To disable alert forwarding for managed clusters, add the `mco-disable-alerting: "true"` annotation to the `MultiClusterObservability` custom resource. When you set the `mco-disable-alerting: "true"` annotation, both the platform and user workload alerts are not forwarded to the Alertmanager on your hub cluster. The forwarding configuration on your managed clusters is reverted.
 
-When the changes are reverted, a ConfigMap named `cluster-monitoring-reverted` is created in the `open-cluster-management-addon-observability` namespace. Any new, manually added alert forward configurations are not reverted from the ConfigMap. 
+Configuration updates to the `ocp-monitoring-config` config map in the `openshift-monitoring` namespace are reverted. Setting the annotation ensures that the `ocp-monitoring-config` config map is not managed or updated by the Observability operator endpoint. After you update the configuration, both platform and user workload Prometheus instances on your managed cluster restarts. Complete the foll ...
```
The configmap name is `cluster-monitoring-config` and not `ocp-monitoring-config`. This seems like a oversight from previous releases too. Apologies for not catching this earlier @dockerymick

dockerymick replied:
No worries, thanks for the info. I will also do that for previous versions (until 2.10)

subbarao-meduri reviewed (approved):
/lgtm

dockerymick commented on observability/observability_alerts.adoc:
```diff
@@ -170,25 +189,51 @@ global:
+To disable alert forwarding for managed clusters, add the `mco-disable-alerting: "true"` annotation to the `MultiClusterObservability` custom resource. When you set the `mco-disable-alerting: "true"` annotation, both the platform and user workload alerts are not forwarded to the Alertmanager on your hub cluster. The forwarding configuration on your managed clusters is reverted.
 
-When the changes are reverted, a ConfigMap named `cluster-monitoring-reverted` is created in the `open-cluster-management-addon-observability` namespace. Any new, manually added alert forward configurations are not reverted from the ConfigMap. 
+Configuration updates to the `ocp-monitoring-config` config map in the `openshift-monitoring` namespace are reverted. Setting the annotation ensures that the `ocp-monitoring-config` config map is not managed or updated by the Observability operator endpoint. After you update the configuration, both platform and user workload Prometheus instances on your managed cluster restarts. Complete the foll ...
```
```suggestion
Configuration updates to the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace are reverted. Setting the annotation ensures that the `cluster-monitoring-config` config map is not managed or updated by the Observability operator endpoint. After you update the configuration, both platform and user workload Prometheus instances on your managed cluster restarts. Complete the following steps:
```

dockerymick committed:
Update observability/observability_alerts.adoc

jc-berger reviewed (approved):
/lgtm

dockerymick commented on observability/observability_alerts.adoc:
```diff
@@ -1,20 +1,39 @@
 = Managing alerts
 
-Receive and define alerts for the observability service to be notified of hub cluster and managed cluster changes.
+Configure Observability alerts to have the following capabilities: 
+
+- Monitor both your hub cluster and managed cluster activity.
+-  Forward platform and user workload alerts to the Alertmanager on your hub cluster.
+-  Route notifications to external systems such as Slack, email, or PagerDuty.
```
Changed this bc it appeared as a long bullet list
```suggestion
Configure Observability alerts to monitor both your hub cluster and managed cluster activity, forward platform and user workload alerts to the Alertmanager on your hub cluster, and route notifications to external systems such as Slack, email, or PagerDuty.
```

dockerymick committed:
Update observability/observability_alerts.adoc

dockerymick committed:
Update observability_alerts.adoc

dockerymick commented on observability/observability_alerts.adoc:
```diff
@@ -1,20 +1,34 @@
-Receive and define alerts for the observability service to be notified of hub cluster and managed cluster changes.
+Configure Observability alerts to monitor both your hub cluster and managed cluster activity, forward platform and user workload alerts to the Alertmanager on your hub cluster, and route notifications to external systems such as Slack, email, or PagerDuty.
 
 - <<alert-prereqs,Prerequisites>>
```
```suggestion
Continue reading to learn more:

- <<alert-prereqs,Prerequisites>>
```

dockerymick committed:
Update observability/observability_alerts.adoc

dockerymick commented:
Thanks for the reviews and approvals!
